plugins/modules/ipanetgroup.py

```diff
@@ -157,18 +157,25 @@
 
 from ansible.module_utils.ansible_freeipa_module import \
     IPAAnsibleModule, compare_args_ipa, gen_add_del_lists, \
-    gen_add_list, gen_intersection_list, ipalib_errors, ensure_fqdn
+    gen_add_list, gen_intersection_list, ensure_fqdn
 
 
 def find_netgroup(module, name):
     """Find if a netgroup with the given name already exist."""
-    try:
-        _result = module.ipa_command("netgroup_show", name, {"all": True})
-    except ipalib_errors.NotFound:
-        # An exception is raised if netgroup name is not found.
-        return None
-    else:
-        return _result["result"]
+    _args = {
+        "all": True,
+        "cn": name,
+    }
+
+    _result = module.ipa_command("netgroup_find", name, _args)
```

Comment:
Please add a comment here why netgroup_find is used and explain how netgroup_show is failing. At best also with a reference to the FreeIPA issue.
This will make it easier later on to understand that this should not simply be switched back to netgroup_show.

Reply:
Updated